typings/internalBinding/code_integrity.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface CodeIntegrityBinding {
+    isAllowedToExecuteFile(filePath: string) : boolean;
+    isFileTrustedBySystemCodeIntegrityPolicy(filePath: string) : boolean;
+    isInteractiveModeDisabled() : boolean;
+    isSystemEnforcingCodeIntegrity() : boolean;
+}

```

Comment:
In order for this to work, typings/global.d.ts file need to import this module...